packages/bappo-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bappo-components",
-  "version": "0.1.0-alpha.186",
+  "version": "0.1.0-alpha.187",
```

Comment:
Hi Emil, we normally update the version when we publish and make it a commit of itself so we can easily locate the version changes when we look at git history.

Reply:
Hi William, thanks for referring me to Jon's write-up. I've reverted the version change.